contracts/src/v0.8/l2ep/dev/scroll/ScrollSequencerUptimeFeed.sol

```diff
@@ -96,6 +98,10 @@ contract ScrollSequencerUptimeFeed is
 
   /// @notice internal method that stores the L1 sender
   function _setL1Sender(address to) private {
+    if (to == address(0)) {
+      revert ZeroAddress();
```

Comment:
Would it make sense to reuse the `InvalidSender` error here?

Reply:
I think it mainly comes down to preference here -

The `InvalidSender` error looks like it was meant for a very specific scenario involving `msg.sender`, so I figured re-using it in other places could be a bit misleading for someone trying to debug or perform error handling

With that in mind, I created a separate error, but I'm open to changing it to the way you described above if you have a strong preference here

Reply:
I see your point. My suggestion was related to how can the name be used to debug. For example if I see just the error name displayed in an event log I wouldn't know which address is this referring to. Maybe we can still create a new error but with a more descriptive naming?

Reply:
Ah I see your point - I've added a `msg` argument to the `ZeroAddress` error, and specified which address is causing the issue using the parameter. Is this more of what you had in mind?